assignments/prototypes.js

```diff
@@ -7,13 +7,23 @@
   
   Each constructor function has unique properites and methods that are defined in their block comments below:
 */
-  
+
 /*
   === GameObject ===
   * createdAt
   * dimensions
   * destroy() // prototype method -> returns the string: 'Object was removed from the game.'
+  * 
 */
+function GameObject(attributes) {
+  // properties
+  this.createdAt = attributes.createdAt;
+  this.dimensions = attributes.dimensions;
+}
+
+GameObject.prototype.destroy = function() {
+  return `Object was removed from the game`;
+};
```

Comment:
This is meant to print out the name of the object, not the word object. Try `this.name` instead.